appwrite/website
# Add blog post on how to optimize your Appwrite project for cost and performance

## What does this PR do?

- Adds blog post on how to optimize your Appwrite project
- Add blog post assets

## Test Plan

- Run the website and navigate to `/blog/post/how-to-optimize-your-appwrite-project`
- Confirm that blog post exists and that the assets are displayed correctly

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes
